src/app/public/src/modules/layout/layout.component.ts

```diff
@@ -59,4 +69,14 @@ export class StacheLayoutComponent implements OnInit, StacheLayout {
         break;
     }
   }
+
+  public ngAfterViewInit() {
+    this.setMinHeight();
+  }
+
+  public setMinHeight() {
```

Comment:
Could this method be `private` so that we wouldn't have to worry about future breaking changes if we wanted to remove (or rename) it?

Reply:
👍